### ISSUE TITLE
Compatibility issues with redstone pen

### ISSUE THREAD

**Issue by RegulusSource** [Kneelawk/WiredRedstone#32]
The redstone lines of this module will charge each other with the redstone pen;And if there are only two redstone lines (no other blocks are connected), they will remain active unless they are disconnected or a separate redstone line is used

**Comment by Kneelawk** [comment]
Yes, this is why wires -> vanilla redstone power only goes one way. I haven't been able to find a good way to prevent this kind of circular powering situation.

The best work around I've found is simply to put a Diode between the two mods' wires in the direction you want the signal to travel.
